lib/measured/rails/active_record.rb

```diff
@@ -1,9 +1,71 @@
-module Measured::Rails
+module Measured::Rails::ActiveRecord
   extend ActiveSupport::Concern
 
   module ClassMethods
-    def measured
+    def measured(measured_class, *fields)
+      options = fields.extract_options!
+      options = {}.merge(options)
 
+      measured_class = measured_class.constantize if measured_class.is_a?(String)
+
+      raise Measured::Rails::Error, "Class must be an instance of Measured::Measurable" unless measured_class.ancestors.include?(Measured::Measurable)
```

Comment:
If measured_class is not a string for whatever reason (e.g. sym) you're going to get a no method error here right? minor but could confuse someone at some point probably.